src/screens/ProfileScreen/elements/TextWithEmoji.tsx

```diff
@@ -4,12 +4,22 @@ import emojiRegex from 'emoji-regex';
 import {COLORS} from '../../../utils/theme';
 import {fonts} from '../../../utils/fonts';
 
-export function TextWithEmoji({text}: {text: string}) {
+export function TextWithEmoji({
+  text,
+  textStyle,
+  testId
+}: {
+  text: string;
+  textStyle: object;
+  testId: string;
+}) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `textStyle` prop is typed as `object`, which is too broad. It would be better to use a more specific type, such as `TextStyle` from React Native. Also, the `testId` prop should be optional because it's typically used for testing and may not always be provided.
```diff
-  textStyle: object;
-  testId: string;
+  textStyle: TextStyle;
+  testId?: string;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->